docs/versioned/install/installing-kourier.md

```diff
@@ -0,0 +1,220 @@
+---
+audience: administrator
+components:
+  - serving
+function: how-to
+---
+
+# Installing Kourier for Knative
+
+This page walks you through manually installing and customizing Kourier for use with Knative. Kourier is an ingress for Knative Serving. Kourier is a lightweight alternative for the Istio ingress as its deployment consists only of an [Envoy proxy](https://www.envoyproxy.io) and a control plane for it.
+
+## Before you begin
+
+This installation is recommended for Knative installations without Istio installed.
+
+You need:
+
+- A Kubernetes cluster with the Knative Serving component installed.
+
+## Supported Kourier versions
+
+You can view the latest tested Kourier version on the [Kourier releases page](https://github.com/knative-extensions/net-kourier/releases).
+
+## Installing Kourier
+
+1. Install Knative Serving if not already installed:
+
+    ``` bash
+    kubectl apply -f https://github.com/knative/serving/releases/latest/download/serving-crds.yaml
+    kubectl apply -f https://github.com/knative/serving/releases/latest/download/serving-core.yaml
+    ```
+
+1. Install Kourier:
+
+    ```bash
+    kubectl apply -f https://github.com/knative/net-kourier/releases/latest/download/kourier.yaml
+    ```
+
+1. Configure Knative Serving to use the proper `ingress.class`:
+
+    ```bash
+    kubectl patch configmap/config-network \
+      -n knative-serving \
+      --type merge \
+      -p '{"data":{"ingress.class":"kourier.ingress.networking.knative.dev"}}'
+    ```
+
+1. Optional - Set your desired domain (replace `127.0.0.1.nip.io` to your preferred domain):
+
+    ```bash
+    kubectl patch configmap/config-domain \
+      -n knative-serving \
+      --type merge \
+      -p '{"data":{"127.0.0.1.nip.io":""}}'
+    ```
```

Comment:
This is not Kourier-specific, though we should describe how to find Kourier's external IP address (from `install-serving-with-yaml.md`):

```bash
kubectl --namespace kourier-system get service kourier
```

I would put this in a separate section (since we have its own page).  Something like "DNS configuration"